bencher.krun

```diff
@@ -0,0 +1,29 @@
+execfile("luajit.krun", globals()) # inherit settings from the main experiment
+
+MAIL_TO = ["fsfod11@gmail.com"]
+
+PRE_EXECUTION_CMDS = [
+    "sudo systemctl stop apt-daily.timer",
+    "sudo systemctl --runtime disable apt-daily.timer", 
+    "sudo systemctl stop apt-daily-upgrade.timer",
+    "sudo systemctl --runtime disable apt-daily-upgrade.timer",
+    
+    "sudo systemctl stop systemd-timesyncd",
+    "sudo systemctl --runtime disable systemd-timesyncd",
+    "sudo systemctl stop cron",
+]
+
+POST_EXECUTION_CMDS = [
+    "sudo systemctl start systemd-timesyncd || true",
+    "sudo systemctl start cron || true",
+]
+
+def custom_dmesg_whitelist(defaults):
+    return defaults + [
+      "^.*en[0-9a-zA-Z]+: link (down|up)",
+      "^.*en[0-9a-zA-Z]+: Link is (up|down).*",
+      "^.*en[0-9a-zA-Z]+: Flow control is (on|off).*",
+      "^.*en[0-9a-zA-Z]+: EEE is (enabled|disabled).*",
+      "^.*random: crng init done",
+      "^.*apt-daily-upgrade.timer: Adding.*",
```

Comment:
I'm surprise you see lines about the apt stuff you have disabled. This suggests they were not disabled properly. Similarly, RNG initialisation shouldn't happen mid-benchmark.

Did you hack `SYNC_WAIT` or supply a really small `TEMP_READ_PAUSE`?

Reply:
If `apt` is doing stuff during the benchmarks, then the results are compromised, so we would have to try and fix that.

Reply:
No the message is from systemd saying the timer entry had a random offset added to its trigger time

Reply:
OK, now what about the RNG?

Reply:
Thats just that system has accumulated enough random noise for the rng as far as I understand.

Reply:
hrm, sounds fishy

Reply:
Did you test this with `--quick`? I wonder if that is the culprit here...

Reply:
Seems to me like the system isn't fully up before krun is running...

Reply:
The only option i used was hardware reboots passed to your cron script